dashboard/about.md

```diff
@@ -47,44 +47,36 @@ Data for the dashboard is pulled from these sources on Mondays and Tuesdays.
 
 *   **Forecaster**
     
-    A model producing quantile predictions
+    A named model that produces forecasts, e.g., "COVIDhub-ensemble"
     
 *   **Forecast**
     
-    A set of data that includes quantile target variable predictions for a certain horizon, epidemiological week, and location 
+    A set of quantile predictions for a specific target variable, epidemiological week, and location 
```

Comment:
Is the removal of "horizon" here intentional?

Reply:
Yes. The horizon is a function of the forecast date and the epidemiological week.